### ISSUE TITLE
2.5.22294: GameSelectorModel#getDefaultGameRealPath:250 - java.nio.file.FileSystemNotFoundException

### ISSUE THREAD

**Issue by tripleabuilderbot** [triplea-game/triplea#9459]
## User Description
all maps gone...again

## TripleA Version
2.5.22294

## Java Version
11.0.6

## Operating System
Mac OS X

## Memory
Heap utilization statistics [MB]
Used Memory: 14
Free memory: 2033
Total memory: 2048
Max memory: 2048

## Stack Trace
```
Exception: java.nio.file.FileSystemNotFoundException
java.lang.Exception
	at jdk.zipfs/jdk.nio.zipfs.ZipFileSystemProvider.getFileSystem(ZipFileSystemProvider.java:169)
	at jdk.zipfs/jdk.nio.zipfs.ZipFileSystemProvider.getPath(ZipFileSystemProvider.java:155)
	at java.base/java.nio.file.Path.of(Path.java:208)
	at games.strategy.engine.framework.startup.ui.panels.main.game.selector.GameSelectorModel.getDefaultGameRealPath(GameSelectorModel.java:250)
	at games.strategy.engine.framework.startup.ui.panels.main.game.selector.GameSelectorModel.gameUriExistsOnFileSystem(GameSelectorModel.java:220)
	at java.base/java.util.Optional.filter(Optional.java:223)
	at games.strategy.engine.framework.startup.ui.panels.main.game.selector.GameSelectorModel.loadDefaultGameSameThread(GameSelectorModel.java:209)
	at games.strategy.engine.framework.GameRunner.showMainFrame(GameRunner.java:80)
	at java.base/java.lang.Thread.run(Thread.java:834)


```

**Comment by beelee1** [comment]
well bummer  Does it work for a while and then disappear ? I know triplea and mac have had problems in the past

**Comment by beelee1** [comment]
closing v 2.5 should b fixed in 2.6
